{"uuid": "b2aaf232-1fa2-4e1e-9bea-f21b429f109c", "children": ["605886b8-6ab2-4f13-8b1e-5a24daa8742c"], "befores": [{"name": "set_up", "status": "passed", "start": 1568726914618, "stop": 1568726918921}], "afters": [{"name": "set_up::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\_pytest\\fixtures.py\", line 786, in _teardown_yield_fixture\n    next(it)\n  File \"D:\\_python\\themis\\Tests/..\\Base\\base.py\", line 21, in set_up\n    self.driver.close()\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 688, in close\n    self.execute(Command.CLOSE)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 319, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 374, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 397, in _request\n    resp = self._conn.request(method, url, body=body, headers=headers)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\request.py\", line 68, in request\n    **urlopen_kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\request.py\", line 89, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\poolmanager.py\", line 323, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 667, in urlopen\n    **response_kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 600, in urlopen\n    chunked=chunked)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\connectionpool.py\", line 354, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 1239, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 1285, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 1234, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 1026, in _send_output\n    self.send(msg)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\http\\client.py\", line 964, in send\n    self.connect()\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\connection.py\", line 181, in connect\n    conn = self._new_conn()\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\connection.py\", line 159, in _new_conn\n    (self._dns_host, self.port), self.timeout, **extra_kw)\n  File \"c:\\users\\z003d7tu\\appdata\\local\\programs\\python\\python36-32\\lib\\site-packages\\urllib3\\util\\connection.py\", line 70, in create_connection\n    sock.connect(sa)\n"}, "start": 1568726950828, "stop": 1568726952844}], "start": 1568726914618, "stop": 1568726952844}